cli/src/semgrep/meta.py

```diff
@@ -99,7 +99,24 @@ def repo_name(self) -> str:
 
     @property
     def repo_url(self) -> Optional[str]:
-        return get_url_from_sstp_url(os.getenv("SEMGREP_REPO_URL"))
+        env = get_state().env
+        repo_url = os.getenv("SEMGREP_REPO_URL")
+        if not repo_url:
+            # if the repo URL was not explicitly provided, try getting it from git
+            # nosem: use-git-check-output-helper
```

Comment:
could you comment why we're not using git_check_output?

Reply:
@underyx I have no idea tbh -- I just re-used the same approach in `repo_name`, which explicitly has a nosem as well. Hmm.